starfish/intensity_table/intensity_table.py

```diff
@@ -167,7 +167,7 @@ def from_spot_data(
     def get_log(self):
         """Deserialize and return a list of pipeline components that have been applied
          throughout a starfish session to create this Intensity Table"""
-        return loads(self.attrs[LOG])
+        return loads(self.attrs[STARFISH])[LOG]
```

Comment:
What happens if we load an IntensityTable with no log?  Or an intensity table without a starfish section in the attrs field?

Reply:
would throwing an error that says "no log info available" be alright?